cmd/storage_bucket_replication_put.go

```diff
@@ -0,0 +1,85 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/exoscale/cli/pkg/storage/sos"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	storageBucketReplicationPutCmd.Flags().StringP(zoneFlagLong, zoneFlagShort, "", zoneFlagMsg)
+	storageBucketReplicationCmd.AddCommand(storageBucketReplicationPutCmd)
+}
+
+var storageBucketReplicationPutCmd = &cobra.Command{
+	Use:   "put sos://BUCKET file://replication.json",
+	Short: "Put replication configuration",
```

Comment:
i'd say publish instead of the raw Put term in the description

Reply:
fixed in 13ae0813